package.json

```diff
@@ -14,7 +14,7 @@
   },
   "engines": {
     "npm": ">=9.0.0",
```

Comment:
Change it to `"npm": ">=10.0.0"`

Reply:
NPM version is closely tied to the Node.js version, so with node `v20.8.0` npm has to be on version major then `10.0.0`. Update `package.json` please
```
{
  ...
  "engines": {
    "npm": ">=10.0.0",
    "node": ">=20.8.0"
  },
  ...
}
```